openshift/dgraph-ratel.yaml

```diff
@@ -1,43 +1,83 @@
----
-apiVersion: v1
-kind: Service
+apiVersion: template.openshift.io/v1
+kind: Template
 metadata:
-  name: ratel-public
+  name: dgraph-ratel-deployment
+  annotations:
+    description: "Dgraph Ratel deployment"
+    openshift.io/display-name: "Dgraph"
+    version: 0.1.0
+    tags: dgraph,zero,alpha,thoth
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: "This is an OpenShift Template to deploy Ratel"
+    template.openshift.io/provider-display-name: "Red Hat, Inc."
   labels:
+    template: dgraph-ratel-deployment
     app: dgraph
-    component: ratel
-spec:
-  type: LoadBalancer
-  ports:
-  - port: 8000
-    targetPort: 8000
-    name: ratel-http
-  selector:
-    app: dgraph
-    component: ratel
----
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: ratel
-  labels:
-    app: dgraph
-    component: ratel
-spec:
-  selector:
-    matchLabels:
-      app: dgraph
-      component: ratel
-  template:
+
+parameters:
+  - description: Hostname to be used for exposing Ratel UI"
+    displayName: Hostname
+    required: true
+    name: RATEL_HOST_NAME
+    value: "ratel-public-thoth-test-core.cloud.paas.psi.redhat.com"
```

Comment:
We could create a "pretty hostname" for this not to have cluster specific this. Added to https://trello.com/c/3DrBN5V2/